### PR TITLE
Added Raspberry Docker

### PR DESCRIPTION
**Pull request by xiazhvera**
*Issue #, if available:*

*Description of changes:*
- Added the docker image for Raspberry Bullseye. Both images are built from Raspbian Stretch based on https://github.com/navikey/raspbian-bullseye

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/docker-images/raspbian-bullseye/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Upgrade to buster from stretch
+###############################################################################
+
+RUN apt-get update \
+	&& apt-get upgrade -y \
+	&& apt-get full-upgrade -y \
+	&& apt-get autoremove --purge -y \
+	&& apt-get clean -y \
+	# Switch to Buster repository.
+	&& sed -i 's/stretch/bullseye/g' /etc/apt/sources.list \
```
Comment (graebm) [inline .github/docker-images/raspbian-bullseye/Dockerfile:13]: multiple comments mention "buster" but code upgrades to "bullseye"

**Review by graebm — APPROVED** [review]
fix & ship

File: .github/docker-images/raspbian-bullseye/Dockerfile (hunk 1 above)
Comment (xiazhvera) [reply .github/docker-images/raspbian-bullseye/Dockerfile]: Thanks!!

File: .github/workflows/create-channel.yml [hunk 2]
```diff
@@ -78,6 +78,7 @@ jobs:
         - rhel8-x64
         - opensuse-leap
         - fedora-34-x64
+        - raspbian-bullseye
```
Comment (bretambrose) [inline .github/workflows/create-channel.yml:81]: Should be added to create-release as well.

**Pull request merged**